frontend/src/App.tsx

```diff
@@ -255,6 +256,7 @@ function App() {
               <CssBaseline />
               <TopBar />
               <Sidebar />
+              <AlertNotification/>
```

Comment:
The mockups show the bar in the central area:
![Screenshot from 2020-11-06 22-15-49](https://user-images.githubusercontent.com/1029635/98419966-124df880-207e-11eb-9e3a-2c15b937badb.png)

Trying this PR it shows on top of the whole view. I don't dislike it in a global position, and I know it's also simpler to execute this way but it will show on top of the cluster button. And if we're having a problem in just one cluster, changing to a different one will be a bit more difficult.